docs/source/DeveloperGuide/AddingNewComponents/DevelopingCommands/dirac-my-great-script.py

```diff
@@ -17,11 +17,9 @@
 __RCSID__ = '$Id$'
 
 from DIRAC import S_OK, S_ERROR, gLogger, exit as DIRACExit
+from DIRAC.Core.Utilities.DIRACScript import DIRACScript
```

Comment:
As this is an example script, you may want to add somewhere a comment reminding about adding it to the `setup.cfg`

Reply:
ef26bd1